### PR TITLE
Added loader to comments

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Added loader to comments on a 'Single news' page, so that user can be sure whether there will be some comments loaded or not.

image

### CODE REVIEW

**Review by yulkats — APPROVED** [review]

**Review by HryshkoAnton — APPROVED** [review]

File: src/app/component/comments/components/add-comment/add-comment.component.html [hunk 1]
```diff
@@ -34,9 +34,16 @@
...
   </form>
 </div>
-<app-comment-body [elements]="elements"
-                  [type]="dataSet.type"
-                  *ngIf="(this.dataSet.type === 'reply' && this.repliesVisibility)
+<div class="comments-loader"
+     *ngIf="elements.length < 1 && elementsArePresent; else comments"
+>
+  <mat-spinner></mat-spinner>
+</div>
```
Comment (mplesha) [inline src/app/component/comments/components/add-comment/add-comment.component.html:41]: Why did you use this unnecessary wrapping `div`? You can use `*ngIf` and `class` to the component `mat-spinner` directly!

File: src/app/component/comments/comments.module.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import { LikesCounterComponent } from './components/likes-counter/likes-counter.
 import { CommentCounterComponent } from './components/comment-counter/comment-counter.component';
 import { CommentPaginationComponent } from './components/comment-pagination/comment-pagination.component';
 import { NgxPaginationModule } from 'ngx-pagination';
+import { MatProgressSpinnerModule } from '@angular/material';
```
Comment (mplesha) [inline src/app/component/comments/comments.module.ts:16]: -- `angular` import between components.
So, organize imports:
order `angular` imports first, next other third party libs and only then imports of your application.

**Comment by mplesha** [comment]
this code has some conflicts

File: src/app/component/comments/comments.module.ts (hunk 2 above)
Comment (RomanDeveloperAcc) [reply src/app/component/comments/comments.module.ts]: Fixed

**Comment by mplesha** [comment]
Build Failed

**Review by sofiiamatsyhin — APPROVED** [review]

File: src/app/component/comments/components/add-comment/add-comment.component.html (hunk 1 above)
Comment (RomanDeveloperAcc) [reply src/app/component/comments/components/add-comment/add-comment.component.html]: Fixed

**Comment by RomanDeveloperAcc** [comment]
> Build Failed

Fixed

**Pull request merged**